packages/linear-genome-view/src/BasicTrack/util/serverSideRenderedBlock.ts

```diff
@@ -203,7 +204,13 @@ function renderBlockData(self: Instance<BlockStateModel>) {
       sequenceConfig = getSnapshot(trackAssemblyData.sequence.adapter)
     }
     const adapterConfig = getConf(track, 'adapter')
-    const adapterConfigId = jsonStableStringify(adapterConfig)
+    // Only subtracks will have parent tracks with configs
```

Comment:
this does assume that the parent track has it's own adapter. should be ok for now